manifests/puppet.pp

```diff
@@ -43,11 +43,11 @@
     pubkey { $ssl_ca_cert:
       key_pair => $::certs::server_ca,
     } ~>
-    file { $client_key:
-      ensure => file,
-      owner  => '52',
-      mode   => '0400',
+    file { [$client_cert, $client_key, $ssl_ca_cert]:
+      ensure  => file,
+      owner   => 'puppet',
+      mode    => '0400',
+      require => Class['puppet::server::install'],
```

Comment:
Could we pull this change out into its own issue then? If we think users of 3.3 are going to need this.